packages/core/core/src/migrations/database/5.0.0-discard-drafts.ts

```diff
@@ -62,13 +133,20 @@ const migrateUp = async (trx: Knex, db: Database) => {
       continue;
     }
 
+    /**
+     * Create plain draft entries for all the entries that were published.
+     */
+    await copyPublishedEntriesToDraft({ db, trx, uid: meta.uid });
+
```

Comment:
you will need to do a 1st pass on all contenttypes to do the copyPublishedEntries 1st before running discardDraft on them as a 2nd pass :)